arch/loongarch/adler32_lasx.c

```diff
@@ -0,0 +1,163 @@
+/* adler32_lasx.c -- compute the Adler-32 checksum of a data stream based on AVX2
+ * Copyright (C) 1995-2011 Mark Adler
+ * Copyright (C) 2022 Adam Stylinski
+ * Authors:
+ *   Brian Bockelman <bockelman@gmail.com>
+ *   Adam Stylinski <kungfujesus06@gmail.com>
+ * Copyright (C) 2025 Vladislav Shchapov <vladislav@shchapov.ru>
```

Comment:
This line should be moved above the authors list, so all copyright lines are grouped before authors list.